R/geom-segment.r

```diff
@@ -42,6 +43,18 @@
 #'     arrow = arrow(length = unit(0.1,"cm"))) +
 #'   borders("state")
 #'
+#' # Use lineend and linejoin to change the style of the segments
+#' df2 <- expand.grid(lineend = c('round', 'butt', 'square'),
+#'                    linejoin = c('round', 'mitre', 'bevel'), stringsAsFactors = FALSE)
+#' df2 <- data.frame(df2, y = 1:9)
+#' ggplot() +
```

Comment:
Can you please use house indenting rules? (http://style.tidyverse.org/syntax.html#long-lines)